ui/component/or-mwc-components/src/or-mwc-input.ts

```diff
@@ -1007,7 +1007,7 @@ export class OrMwcInput extends LitElement {
                                      aria-expanded="false"
                                      aria-disabled="${""+(this.disabled || this.readonly)}"
                                      aria-labelledby="label selected-text">
-                                    <span class="mdc-select__ripple"></span>
```

Comment:
Isn't it weird to put undefined there (why not just if !outlined put html there? no else)? Or is that common practice?

Reply:
Used it myself in multiple places, for example the dashboard builder. Would imagine leaving it `undefined` would render nothing instead of an empty html, which would be uneccessairy. Had no issues using it yet.